### ISSUE TITLE
[FEATURE REQUEST] Hide packages/bundles(/apps) from search result

### ISSUE THREAD

**Issue by Revadike** [tfedor/AugmentedSteam#1120]
### Motivation / Use Case for Changing the Behavior
The search results in the steam store show a mixed bag of apps, packages and bundles. I'm mostly interested in apps. So an option to hide it would be fantastic.

### Current Behavior (if any)
The search results in the steam store show a mixed bag of apps, packages and bundles.

### Desired Behavior
Only show the type you want to see, by hiding other types.

**Comment by candela97** [comment]
Does this do what you want?
![螢幕擷取畫面 2021-05-22 053450](https://user-images.githubusercontent.com/54083835/119204392-e5b47800-bac7-11eb-9e8c-70594b6eefbe.jpg)

I not exactly sure how the "Include Bundles" option works though...

**Comment by Revadike** [comment]
Sadly not, you'd expect that to work, but Steam works in mysterious ways and still includes packages in search results.

**Comment by candela97** [comment]
Guess it only covers `https://store.steampowered.com/bundle/` links. I think making this option also show/hide rows whose link starts with `https://store.steampowered.com/sub/` would work.

**Comment by kirara** [comment]
SteamDB has it's own caveats. It has 2 useful searches. One searches only through the current deals and you can't filter out VR Only / Adult Only / Non-english games (my usual use case). You can do this in instant search where you can't sort by anything else than rating, you can't filter by number of ratings, you can't filter owned / wishlisted / ignored games. There's also no infinite scroll in neither of those. I'd say i still like the OG Steam search the most, but this issue bothers me as well.

You don't have to parse the URL, you can easily filter `[data-ds-packageid]` selector.

**Comment by MxtOUT** [comment]
Ok, I see how it can be used, but I really don't want to have three more checkboxes there.
Splitting the existing "Augmented Steam filters" block up and distributing the filters to the other blocks could help.

**Comment by Korb** [comment]
> Guess it only covers `https://store.steampowered.com/bundle/` links. I think making this option also show/hide rows whose link starts with `https://store.steampowered.com/sub/` would work.

Indeed, now in Steam Search there is no way to hide links like `https://store.steampowered.com/sub/`. Neither with standard tools, nor with Augmented Steam 2.1.0. I join the suggestion to add this filter.

**Comment by MxtOUT** [comment]
> If I may.... Even if it's messing with the default behavior, the expected and most intuitive behavior would be to show only apps as default (i.e. exclude packages). It already doesn't show bundles by default and also despite its weird label you can show ONLY bundles by selecting only Include Bundles. This way you could add single checkbox to also "Include Packages" (if we'd go with the naming convention) to not exclude them AND show them exclusively when its the only thing selected. That would cover all the cases.

I'm not a fan of hiding default search results, so this has to be solved differently.

Steam shows bundles by default, just scroll down far enough on an empty search page or take a look at this search: https://store.steampowered.com/search/?term=bundle

The "Include Bundles" button is just weirdly worded because other toggles in the "Show selected types" category are just simple nouns.
Remove the "Include" and things become way clearer.

Steam has a way to exclude bundles via the "Games" toggle in "Show selected types", as @candela97 pointed out.
We could fix this functionality by also excluding packages and add another "Packages" option to the "Show selected types" category.
Although this would be an intervention in default behavior (since we change what the "Games" toggle does) I think it is okay given that the user actively has to select this option.
